drivers/src/sha384acc.rs

```diff
@@ -24,6 +24,13 @@ const MAX_MAILBOX_CAPACITY_BYTES: u32 = 128 << 10;
 
 pub type Sha384Digest<'a> = &'a mut Array4x12;
 
+#[repr(u32)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum ShaAccLockState {
```

Comment:
Since this is in the sha384acc module, it would be ok to call this AssumedLockState, which I think makes it clearer what is actually happening here.

Reply:
Updated

Reply:
I see you updated the variable name. Did you intend to update the type name too? If you prefer it this way, that's fine too...